README.md

```diff
@@ -19,6 +25,38 @@ Run `npm start` for a dev server. Navigate to `http://localhost:4200/`. The app
 
 Run `npm run test` to execute the unit tests via Jest
 
-## Commit
+## Technologies
+
+1. Angular
+2. Typescript
+3. RxJS
+4. [D3](https://d3js.org/)
+5. [Spectator](https://github.com/ngneat/spectator) (_Unit Testing_)
+
+## UI Architecture
+
+| <img src="https://hypertrace-docs.s3.amazonaws.com/ui-architecture.png" width="400" height="400"/> |
```

Comment:
We should probably update this diagram, it's outdated from before the HT breakout. but it'll work temporarily.

Reply:
cool. we shall update in next week's revision probably.